docs/modules/ROOT/pages/main/development.adoc

```diff
@@ -2,28 +2,30 @@
 
 == Development
 
-The modelix project is developed open-source under the https://www.apache.org/licenses/LICENSE-2.0.html[Apache-2.0 license].
-All components and software of the platform as well as issue tracking and project management are done in repositories in our https://github.com/modelix[GitHub organisation].
+The modelix project is developed open-source under the https://www.apache.org/licenses/LICENSE-2.0.html[Apache-2.0 license^].
```

Comment:
I hate it when pages try to decide for me whether I want a new tab or not. I can easily select that on my own with either clicking the LMB or the wheel

Reply:
> Configuring a link that points to a location outside the current site is common practice to avoid disrupting the reader’s flow.

(information comes from same page as above)

Seems to be a common practice...

Reply:
I've seen that. But still, sometime I just want to follow in the same tab and that things makes this impossible to me. If not specified, I could freely decide.